syllabuspdf/apirequest.py

```diff
@@ -16,6 +21,7 @@ def fetch_syllabus(id):
 	'''Fetches syllabus for course with given id from API. Returns syllabus as HTML string'''
 	if oauthtoken is None:
 		raise Exception("Invalid Authorization Token")
+		logger.warning("Missing oauthtoken")
```

Comment:
This log statement should go before the exception is raised, otherwise the log statement will never get executed. Also, let's add a helpful hint to the log message so that if another developer sees the message in the logs, they'll know where to look. Maybe something like this:

```python
logger.warning("Missing oauthtoken. Check that `oauthtoken` is defined in SECURE_SETTINGS file.")
```